Services/Dashboard/ItemsBlock/classes/class.ilPDSelectedItemsBlockViewSettings.php

```diff
@@ -291,6 +301,62 @@ public function getActivePresentationsByView(int $view): array
             : unserialize($val, ['allowed_classes' => false]);
     }
 
+    public function setViewPositions(array $positions): void
```

Comment:
Please add the doc type for the `$position` parameter. In general: If it is not specified in a parent class or interface, the type of an array parameter or return value should be written in the doc block.
If there are other instances where the doc block is missing, please add them accordingly.